pact-jvm-model/src/test/groovy/au/com/dius/pact/model/PactSerialiserSpec.groovy

```diff
@@ -39,12 +39,12 @@ class PactSerialiserSpec extends Specification {
     provider = new Provider('test_provider')
     consumer = new Consumer('test_consumer')
     def requestMatchers = new MatchingRulesImpl()
-    requestMatchers.addCategory('body').addRule('$.test', new TypeMatcher())
+    requestMatchers.addCategory('body').addRule('$.test', TypeMatcher.INSTANCE)
```

Comment:
calling private constructor of TypeMatcher was breaking equality test. Strange thing is that it worked before ;)

Reply:
oh, yeah, that is wrong. TypeMatcher is a singleton so the constructor should have never been called, but you can call private things in groovy.